identifiers/country-au.csv

```diff
@@ -1,618 +1,623 @@
-ocd-division/country:au/territory:act/federal_electorate:bean,Australian Capital Territory - Federal House of Representatives Bean electorate,2018-07-13,
-ocd-division/country:au/territory:act/federal_electorate:canberra,Australian Capital Territory - Federal House of Representatives Canberra electorate,,
-ocd-division/country:au/territory:act/federal_electorate:fenner,Australian Capital Territory - Federal House of Representatives Fenner electorate,2016-01-28,
-ocd-division/country:au/territory:act/federal_electorate:fraser,Australian Capital Territory and Jarvis Bay Territory - Federal House of Representatives Fraser electorate,,2016-01-27
-ocd-division/country:au/territory:act/sldl:brindabella,Brindabella,,
-ocd-division/country:au/territory:act/sldl:ginninderra,Ginninderra,,
-ocd-division/country:au/territory:act/sldl:kurrajong,Kurrajong,,
-ocd-division/country:au/territory:act/sldl:murrumbidgee,Murrumbidgee,,
-ocd-division/country:au/territory:act/sldl:yerrabi,Yerrabi,,
-ocd-division/country:au/territory:jbt,Jervis Bay Territory,,
-ocd-division/country:au/territory:nt,Northern Territory,,
-ocd-division/country:au/territory:nt/federal_electorate:lingiari,Northern Territory - Federal House of Representatives Lingiari electorate,,
-ocd-division/country:au/territory:nt/federal_electorate:solomon,Northern Territory - Federal House of Representatives Solomon electorate,,
-ocd-division/country:au/territory:nt/sldl:arafura,Arafura,,
-ocd-division/country:au/territory:nt/sldl:araluen,Araluen,,
-ocd-division/country:au/territory:nt/sldl:arnhem,Arnhem,,
-ocd-division/country:au/territory:nt/sldl:barkly,Barkly,,
-ocd-division/country:au/territory:nt/sldl:blain,Blain,,
-ocd-division/country:au/territory:nt/sldl:braitling,Braitling,,
-ocd-division/country:au/territory:nt/sldl:brennan,Brennan,,
-ocd-division/country:au/territory:nt/sldl:casuarina,Casuarina,,
-ocd-division/country:au/territory:nt/sldl:daly,Daly,,
-ocd-division/country:au/territory:nt/sldl:drysdale,Drysdale,,
-ocd-division/country:au/territory:nt/sldl:fannie_bay,Fannie Bay,,
-ocd-division/country:au/territory:nt/sldl:fong_lim,Fong Lim,,
-ocd-division/country:au/territory:nt/sldl:goyder,Goyder,,
-ocd-division/country:au/territory:nt/sldl:gwoja,Gwoja,,
-ocd-division/country:au/territory:nt/sldl:johnston,Johnston,,
-ocd-division/country:au/territory:nt/sldl:karama,Karama,,
-ocd-division/country:au/territory:nt/sldl:katherine,Katherine,,
-ocd-division/country:au/territory:nt/sldl:mulka,Mulka,,
-ocd-division/country:au/territory:nt/sldl:namatjira,Namatjira,,
-ocd-division/country:au/territory:nt/sldl:nelson,Nelson,,
-ocd-division/country:au/territory:nt/sldl:nightcliff,Nightcliff,,
-ocd-division/country:au/territory:nt/sldl:port_darwin,Port Darwin,,
-ocd-division/country:au/territory:nt/sldl:sanderson,Sanderson,,
-ocd-division/country:au/territory:nt/sldl:spillett,Spillett,,
-ocd-division/country:au/territory:nt/sldl:wanguri,Wanguri,,
+id,name,sameAs,sameAsNote,validThrough,validFrom,validTo

```

Comment:
It looks like we had erroneously used `validTo` previously. Could you update the `federal_electorates.csv` header to use `ValidThrough` instead? This will allow the columns to properly merge here

Reply:
Hi @jloutsenhizer , we have committed the suggested changes in aliases.csv and federal_electorates.csv. Please let us know if any other changes are required.

Reply:
Thanks, lgtm!